locator/tablets.cc

```diff
@@ -269,6 +269,10 @@ const tablet_map& tablet_metadata::get_tablet_map(table_id id) const {
     }
 }
 
+bool tablet_metadata::has_tablet_map(table_id id) const {
```

Comment:
nit: for bisectability reasons, move the declaration and definition of this function to the previous patch (`load_sketch: Tolerate missing tablet_map when selecting for a given table`).

Otherwise, LGTM

Reply:
Right